README.md

```diff
@@ -33,12 +33,6 @@ Run npm start in your local browser or using Live Server in Visual Studio Code.
 - GitHub: (https://github.com/rotshidzw)
 - Twitter: (https://twitter.com/footballlz )
 - LinkedIn: (https://www.linkedin.com/in/rochidzwa-chester-8062b6211/)
--
```

Comment:
- [ ] Good job so far 🙌🏼. Please make sure you record max 5 mins video presenting the feature of your project 📹